apps/demos/Demos/Chat/AIIntagration/description.md

```diff
@@ -0,0 +1 @@
+Chat is an interactive interface that allows users to send and receive messages in real time.
```

Comment:
The task in CI fails if we mix changes in code and markdown files. It's better to separate these types of changes

Reply:
This task is optional and can be skipped. DevOps told me how it works: if you're a technical writer and you only make changes to .md files, you don't have to wait for all the other tests to turn green.